Support JSX fragments with jsxFragFactory compiler option and @jsxFrag pragma (#35392)
microsoft/TypeScript: tests/baselines/reference/inlineJsxFactoryWithFragmentIsError.errors.txt

    import * as React from "./renderer";
    <><h></h></>
    ~~~~~~~~~~~~

> !!! error TS17017: JSX fragment needs corresponding @jsxFrag prama when using @jsx pragma.

Reviewer: ```suggestion
!!! error TS17017: An @jsxFrag pragma is required when using an @jsx pragma with JSX fragments.
```
